style.css

```diff
@@ -1,8 +1,169 @@
-footer {
-  background-color: blue;
+.nav {
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  border-top: 3px solid #309;
+  border-bottom: 3px solid #309;
+  width: 1440px;
+  max-height: 66px;
+  padding-top: 44px;
+}
+
+.nav__link {
+  font-family: "gilroy", sans-serif;
+  font-style: normal;
+  font-weight: 800;
+  font-size: 20px;
+  color: #585858;
+  cursor: pointer;
+  margin-bottom: 41px;
+  margin-right: 36px;
+  line-height: 25px;
+  text-decoration: none;
+  text-align: center;
+  
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Azez3/pullRequest?prid=6273092)